packages/react/src/components/ThemeProvider/ComponentStyle.tsx

```diff
@@ -1,17 +1,9 @@
 import * as React from 'react';
 import { WebTheme, createComponentCSS } from '@aws-amplify/ui';
-import {
-  BaseComponentProps,
-  ElementType,
-  ForwardRefPrimitive,
-  Primitive,
-  PrimitiveProps,
-} from '../../primitives/types';
-import { primitiveWithForwardRef } from '../../primitives/utils/primitiveWithForwardRef';
 import { BaseComponentTheme } from '@aws-amplify/ui';
 import { Style } from './Style';
 
-interface BaseComponentStyleProps extends BaseComponentProps {
+interface ComponentStyleProps extends React.StyleHTMLAttributes<{}> {
```

Comment:
Same thought as [here](https://github.com/aws-amplify/amplify-ui/pull/5825/files#diff-43aa10f06463990535af191d9d2710c6b14c2cc55810905f4bc5444cf9c6dde2R5)

```suggestion
interface ComponentStyleProps extends React.StyleHTMLAttributes<'style'> {
```